apps/src/code-studio/components/IconList.jsx

```diff
@@ -68,3 +66,4 @@ module.exports = IconList;
 
 window.dashboard = window.dashboard || {};
 window.dashboard.IconList = IconList;
+window.dashboard.iconsUnicode = require('./icons').unicode;
```

Comment:
It looks like the only usage of this is in apps, such that we probably dont need to stick this on dashboard anymore https://github.com/code-dot-org/code-dot-org/blob/44cfd1ceeeb55e0b5979e316ebbecd7a5f0fcd11/apps/src/assetManagement/assetPrefix.js#L84